### PR TITLE
Improving transtation

### PR DESCRIPTION
**Pull request by sergekhakhin**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -1878,7 +1878,7 @@ msgstr "&Информация"
 
 #: dnfdragora/ui.py:605
 msgid "&History"
-msgstr ""
+msgstr "История"
```
Comment (yurchor) [inline po/ru.po:1881]: Please keep accelerators in place. Somebody might want to use them.

File: po/ru.po [hunk 2]
```diff
@@ -1838,7 +1838,7 @@ msgstr "в именах файлов"
 
 #: dnfdragora/ui.py:551
 msgid "Use regexp"
-msgstr ""
+msgstr "Использовать regexp"
```
Comment (yurchor) [inline po/ru.po:1841]: AFAIK, "regexp" is "регулярное выражение" in Russian.

**Review by yurchor — CHANGES_REQUESTED** [review]

File: po/ru.po (hunk 2 above)
Comment (sergekhakhin) [reply po/ru.po:1841]: Yes, it is but wouldn't it be too clunky to have a checkbox three times longer or it's a common thing. I don't get to use Russian locale in software much, so I'm just asking.

File: po/ru.po (hunk 1 above)
Comment (sergekhakhin) [reply po/ru.po:1881]: Yeah, that's for sure.

File: po/ru.po (hunk 2 above)
Comment (yurchor) [reply po/ru.po:1841]: It can be cut off without losing meaning. Just get rid of the "Use" translation.

**Comment by yurchor** [comment]
Two messages were updated according to the style of the rest of the translation. Closing this MR.

**Pull request closed without merging**